src/views/product-landing/helpers/validate-against-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { Schema, ValidationErrorItem } from 'joi'
+
+export function validateAgainstSchema<T>(
```

Comment:
Potentially re-usable, but didn't want to pre-emptively elevate, so have it as a view `helper` for now 👍